compose/install.md

```diff
@@ -117,6 +117,78 @@ on Microsoft Windows Server and want to install Docker Compose.
 
 ### Install Compose on Linux systems
 
+You can install Docker Compose in different ways, depending on your needs:
+
+- In testing and development environments, some users choose to use automated
+  [convenience scripts](#install-using-the-convenience-script) to install Docker.
+
+- Most users
+  [set up Docker's repositories](#install-using-the-repository) and install
+  from them, for ease of installation and upgrade tasks. This is the
+  recommended approach.
+
+- Some users download the binary and
+  [install it manually](#install-binary-manually) and manage
+  upgrades completely manually.
+
+
+#### Install using the convenience script
+
+As Docker Compose is now part of the Docker CLI it can be installed via a convenience script with Docker Engine and the CLI.  
+[Choose your Linux distribution](../engine/install/index.md#server) and follow the instructions.
+</div>
+<div id="linux-standalone" class="tab-pane fade" markdown="1">
+
+
+#### Install using the repository
+
+If you already follow the instructions to install Docker Engine, Docker Compose should already be installed.   
+Otherwise, you can set up the Docker repository as mentioned in the Docker Engine installation, [choose your Linux distribution](../engine/install/index.md#server) and go to the `Set up the repository` section.
+
+When finished
+
+1. Update the `apt` package index, and install the _latest version_ of Docker Compose, or go to the next step to install a specific version:
+
+    ```console
+    $ sudo apt-get update
+    $ sudo apt-get install docker-compose-plugin
+    ```
+
+   > Got multiple Docker repositories?
+   >
+   > If you have multiple Docker repositories enabled, installing
+   > or updating without specifying a version in the `apt-get install` or
+   > `apt-get update` command always installs the highest possible version,
+   > which may not be appropriate for your stability needs.
```

Comment:
I don't think we publish pre-releases for compose, so I suggest we can remove this note to slightly reduce the noise;

```suggestion
```